README.md

```diff
@@ -31,6 +31,8 @@ was written correctly, and much more. It can also directly flash Raspberry Pi de
 Refer to the [downloads page][etcher] for the latest pre-made
 installers for all supported operating systems.
 
+> Note: Our deb|rpm packages are now hosted on [Cloudsmith](https://cloudsmith.com)!
```

Comment:
Perhaps "deb and rpm" is clearer than "deb|rpm" ? :shrug:

Reply:
Good point, looks better